api_tests/e2e/rfc003/eth-erc20_btc/happy.js

```diff
@@ -1,26 +1,32 @@
 const chai = require("chai");
 chai.use(require("chai-http"));
 const wallet = require("../../../lib/wallet.js");
-const Web3 = require("web3");
+const utils = require("web3-utils");
 const ethereum = require("../../../lib/ethereum.js");
 const bitcoin = require("../../../lib/bitcoin.js");
 const actor = require("../../../lib/actor.js");
 const should = chai.should();
 
-const toby_wallet = wallet.create("toby");
+const toby_wallet = wallet.create("toby", {
+    ethConfig: global.harness.ledgers_config.ethereum,
+});
 
-const toby_initial_eth = "10";
-const alice_initial_eth = "5";
-const alice_initial_erc20 = Web3.utils.toWei("10000", "ether");
+const toby_initial_eth = utils.toBN(10);
+const alice_initial_eth = utils.toBN(5);
```

Comment:
So we're using BN now everywhere rather than BigInt. What's the reason? Ethereum ts only accepts BN?

Reply:
Indeed, Ethereum support BN not BigInt.
Better keep uniform and only use one "Big Number" type.